app/views/layouts/dashboard.html.erb

```diff
@@ -197,56 +197,9 @@
                     </div>
                     <div class="dropdown-divider"></div>
                     <ul class="list-unstyled">
-                      <li class="dropdown-submenu dropstart-lg">
-                        <a class="dropdown-item dropdown-list-group-item dropdown-toggle" href="#">
-                          <i class="fe fe-circle me-2"></i> Status
-                        </a>
-                        <ul class="dropdown-menu">
-                          <li>
-                            <a class="dropdown-item" href="#">
-                              <span class="badge-dot bg-success me-2"></span> Online
-                            </a>
-                          </li>
-                          <li>
-                            <a class="dropdown-item" href="#">
-                              <span class="badge-dot bg-secondary me-2"></span> Offline
-                            </a>
-                          </li>
-                          <li>
-                            <a class="dropdown-item" href="#">
-                              <span class="badge-dot bg-warning me-2"></span> Away
-                            </a>
-                          </li>
-                          <li>
-                            <a class="dropdown-item" href="#">
-                              <span class="badge-dot bg-danger me-2"></span> Busy
-                            </a>
-                          </li>
-                        </ul>
-                      </li>
-                      <li>
-                        <a class="dropdown-item" href="../../pages/profile-edit.html">
-                          <i class="fe fe-user me-2"></i> Profile
-                        </a>
-                      </li>
-                      <li>
-                        <a class="dropdown-item" href="../../pages/student-subscriptions.html">
-                          <i class="fe fe-star me-2"></i> Subscription
-                        </a>
-                      </li>
-                      <li>
-                        <a class="dropdown-item" href="#">
-                          <i class="fe fe-settings me-2"></i> Settings
-                        </a>
-                      </li>
-                    </ul>
-                    <div class="dropdown-divider"></div>
-                    <ul class="list-unstyled">
-                      <li>
-                        <a class="dropdown-item" href="../../index.html">
-                          <i class="fe fe-power me-2"></i> Sign Out
-                        </a>
-                      </li>
+                        <li>
+                          <%= button_to "<i class='fe fe-power me-2'></i> Sign Out".html_safe, destroy_user_session_path, method: :delete, class: 'dropdown-item', form: { data: { turbo: false } } %>
```

Comment:
To avoid having to use `.html_safe` here you could use the button_to and a pass it a block. 
```
<%= button_to url, ... do %>
   <i class='fe fe-power me-2'></i> Sign Out
<% end %>
```
You'll need to double check the syntax.

Reply:
Sorry, I did not mean to assign this for review. 
But thank I will make that change

Reply:
This has been changed